plugins/by-name/neorg/default.nix

```diff
@@ -20,6 +20,14 @@ with lib;
       ];
     };
 
+    neorgTelescopePackage = lib.mkPackageOption pkgs "neorg-telescope" {
```

Comment:
Moved to a plugin module?

Reply:
Eh, looks like it wouldn't really makes sense on its own. https://github.com/nvim-neorg/neorg-telescope?tab=readme-ov-file#-installation